elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metric/MetricUtils.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.metric;
+
+import com.yahoo.elide.core.EntityDictionary;
+import com.yahoo.elide.datastores.aggregation.annotation.MetricAggregation;
+import com.yahoo.elide.datastores.aggregation.annotation.MetricComputation;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Assorted methods for constructing and using {@link Metric} instances
+ */
+@Slf4j
+public final class MetricUtils {
+
+    /**
+     * Constructor.
+     * <p>
+     * Suppress default constructor for noninstantiability.
+     */
+    private MetricUtils() {
+        throw new AssertionError();
+    }
+
+    /**
+     * Returns whether or not an entity field is a simple metric, i.e. a metric field not decorated by
+     * {@link MetricComputation}.
+     *
+     * @param fieldName  The entity field
+     * @param containingClz  The entity containing the field
+     *
+     * @return {@code true} if the field is a metric field and also a base metric without JPQL expression annotated
+     */
+    public static boolean isBaseMetric(String fieldName, Class<?> containingClz, EntityDictionary entityDictionary) {
```

Comment:
This function can be removed if you adopt my other comment on metric construction..  Then we can remove the MetricUtils class.